site/ipam.md

```diff
@@ -22,7 +26,11 @@ You can see which address was allocated with
     host1# weave ps $C
     a7aff7249393 7a:51:d1:09:21:78 10.2.3.1/24
 
-The `-iprange` parameter is given in [CIDR
+To use a subnet other than the default, use `--subnet`:
```

Comment:
"To request the allocation of an address from a subnet other than the default..."